### DISCUSSION TITLE
err why is there a bark fandom

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#130] [Admin General]
fandom sucks, also [we already have a wiki](https://github.com/mariocraft987/bark-coding/wiki)

**Comment by dumorando** [comment]
so real

**Comment by Mrdev88** [comment]
Yeah, fandom sucks that I usually use miraheze or GitHub for wiki, we should stay with the GitHub wiki we have

**Comment by Mrdev88** [comment]
If the admins would allow me, I can convert the wiki to a website

**Comment by mariocraft987** [comment]
Permission granted!

**Comment by mariocraft987** [comment]
Fixed in [`38d7270`](https://github.com/mariocraft987/bark-coding/commit/38d727073f42f002f20d6cc7a991d98e4aa089fe).
